telos-frontend/src/views/journal/Journal.jsx

```diff
@@ -41,27 +52,85 @@ const Journal = () => {
     return new Date(date.getTime() - date.getTimezoneOffset() * 60000).toISOString().split('T')[0];
   }
 
+  const addNewWidget = (widgetName) => {
```

Comment:
This is clean and easy to understand - good job!

Reply:
Thanks! 🙂